File: tests/drivers/csharp/Basic.cs
using System;
using System.Linq;
using Neo4j.Driver.V1;

public class Basic {
  public static void Main(string[] args) {
    var config = Config.DefaultConfig;
    config.EncryptionLevel = EncryptionLevel.None;
    using(var driver = GraphDatabase.Driver("bolt://localhost:7687", AuthTokens.None, config))
      using(var session = driver.Session())
      {
        session.Run("MATCH (n) DETACH DELETE n").Consume();
        session.Run("CREATE (alice:Person {name: \"Alice\", age: 22})").Consume();
        var result = session.Run("MATCH (n) RETURN n").First();
        var alice = (INode) result["n"];
        Console.WriteLine(alice["name"]);
        Console.WriteLine(string.Join(", ", alice.Labels));
        Console.WriteLine(alice["age"]);
      }
    Console.WriteLine("All ok!");
  }
}

File: tests/drivers/go/v4/basic.go
package main

import "github.com/neo4j/neo4j-go-driver/neo4j"
import "fmt"
import "log"

func handle_error(err error) {
  log.Fatal("Error occured: %s", err)
}

func main() {
  configForNeo4j40 := func(conf *neo4j.Config) { conf.Encrypted = false }

  driver, err := neo4j.NewDriver("bolt://localhost:7687", neo4j.BasicAuth("", "", ""), configForNeo4j40)
  if err != nil {
    log.Fatal("An error occurred opening conn: %s", err)
  }

  defer driver.Close()

  sessionConfig := neo4j.SessionConfig{AccessMode: neo4j.AccessModeWrite}
  session, err := driver.NewSession(sessionConfig)
  if err != nil {
    log.Fatal("An error occured while creating a session: %s", err)
  }

  defer session.Close()

  result, err := session.Run("MATCH (n) DETACH DELETE n", map[string]interface{}{})
  if err != nil {
   handle_error(err)
  }
  _, err = result.Consume()
  if err != nil {
    handle_error(err)
  }

  result, err = session.Run(`CREATE (alice:Person {name: "Alice", age: 22})`, map[string]interface{}{})
  if err != nil {
    handle_error(err)
  }
  _, err = result.Consume()
  if err != nil {
    handle_error(err)
  }

  result, err = session.Run("MATCH (n) RETURN n", map[string]interface{}{})
  if err != nil {
    handle_error(err)
  }

  if !result.Next() {
    log.Fatal("Missing result")
  }

  node_record, has_column := result.Record().Get("n")
  if !has_column {
    log.Fatal("Wrong result returned")
  }
  node_value := node_record.(neo4j.Node)

  fmt.Println(node_value.Props()["name"])
  fmt.Println(node_value.Labels())
  fmt.Println(node_value.Props()["age"])

  fmt.Println("All ok!")
}
